plugins/woocommerce/includes/admin/views/html-admin-page-status-report.php

```diff
@@ -177,11 +177,12 @@
 			<td class="help"><?php echo wc_help_tip( esc_html__( 'The maximum amount of memory (RAM) that your site can use at one time.', 'woocommerce' ) ); /* phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped */ ?></td>
 			<td>
 				<?php
-				if ( $environment['wp_memory_limit'] < 67108864 ) {
+				$wp_memory_limit = wc_let_to_num($environment['wp_memory_limit']);
```

Comment:
The linter is complaining about this (missing spaces around the parenthesis) and other similiar lines in this file. Could you please fix them? You can do so by running `vendor/bin/phpcbf includes/admin/views/html-admin-page-status-report.php` while inside the `plugins/woocommerce` directory.